versions.json

```diff
@@ -317,8 +305,42 @@
     },
     "aznfs": {
         "common": {
-            "version": "2.0.9",
-            "sha256": "36d9f34bb0554acad27f77dd2d1a7d1f8a2246a63a670d26ec2db3e680bf838f"
+            "version": "2.0.12",
+            "sha256": "5ad0f1986c4d211d2bd9626b2f246fbcd556c1fb0ca7be346adf660076c49210"
+        }
+    },
+    "rocm": {
```

Comment:
Please use ROCm version 6.2.2 (drop the proposed 6.4.0 version update), and use the latest RCCL version that works in our validation pipeline.